scripts/jenkins-yoctobuild-build.sh

```diff
@@ -651,6 +651,8 @@ add_to_build_list() {
 # Selects a configuration from the list to build. Client builds override
 # server-only builds, but the function otherwise tries to detect whether two
 # configurations conflict.
+INTEGRATION_TEST_SUITE=$($WORKSPACE/integration/extra/release_tool.py --select-test-suite)
```

Comment:
What if the release_tool does not accept such parameter? For instance and old release.

I suggest amending it with a default, something like: `$($WORKSPACE/integration/extra/release_tool.py --select-test-suite | echo "all")`

Reply:
Hehe. The build failures are exactly due to this :)

ref https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/77754277

Reply:
fixed

Reply:
:+1: